### ISSUE TITLE
manifesto = 404

### ISSUE THREAD

**Issue by devbymike** [fetchlogic/buildshiprepeat.com#1]
Link broken to page http://buildshiprepeat.com/manifesto/ - produces 404 error not a manifesto.

**Comment by joshtronic** [comment]
Yeah, we're still pre-launch at the moment :P Will leave this open until we :shipit: though. Thanks!
